app/static/engine-configs/suricata/suricata-6.0.0.yaml

```diff
@@ -1,6 +1,6 @@
 %YAML 1.1
 ---
-
+# You found it!! /opt/dalton/engine-configs/suricata/suricata-6.0.0.yaml
```

Comment:
did this get left in on accident?

Reply:
Yeah, took me a bit of hunting to find where those configs were coming from.  Removed.